content/md/first-steps/manage-reference-entities.md

```diff
@@ -0,0 +1,233 @@
+1.  Go to `Entities` menu
+1.  Click on `Create` button
+1.  Input the label of the reference entity in the current catalog locale
+1.  The code is automatically generated using the label but you can update it (this code must be unique and may only contain letters, numbers and underscores)
+1.  Click on `Save` button
+
+![image](../img/ReferenceEntity_CreateEntity.png)
+
+Then you can [define its properties](/articles/manage-reference-entities.html#define-its-properties) (`Properties` tab) and [define its records attributes](/articles/manage-reference-entities.html#define-its-records-attributes) (`Attributes` tab).
+
+:::info
+To guarantee that the PIM is functional and runs smoothly, we defined a volume limitation of 100 reference entities for this feature. If you try to create more than 100 reference entities, you will have a blocking message.
+:::
+
+# Define its properties
+In `Properties` tab, you can edit the label of the reference entity and upload an image used to identify the reference entity in the PIM.
+
+You can also translate the label of the reference entity by switching the current catalog locale in the header.
+
+![image](../img/ReferenceEntity_EditProperties.png)
+
+# Define its records attributes
+In `Attributes` tab, you can define what are the attributes for the records of this reference entity.  
+For example, a `Brand` reference entity could be described by the following information:
+- a code
+- a label
+- an image
+- a country
+- a designer
+- a foundation date
+- a photo
+- a description
+
+## Add an attribute
+To add an attribute, follow these steps:
+1.  Go to `Attributes` tab for a reference entity
+1.  Click on `Add an attribute` button
+1.  Input the label of the attribute in the current catalog locale
+1.  The code is automatically generated using the label but you can update it (this code must be unique and may only contain letters, numbers and underscores)
+1.  Select the attribute type:
+ ![image](../img/ReferenceEntity_AddAttribute_Type.png)
+    - **Text**: for text or text area attributes
+    - **Reference entity single link**: to link a record to only one entity record
+    - **Reference entity multiple links**: to link a record to several entity records
+    - **Single option**: to define a list of options with a single choice
+    - **Multiple options**: to define a list of options with multiple choices
+1.  If the attribute type is a **reference entity single/multiple link**, select the reference entity to link to this attribute in the list
+1.  Tick `Value per locale` if your record could have different values per locale for this attribute
+1.  Tick `Value per channel` if your record could have different values per channel for this attribute
+1.  Click on `Save` button  
+
+The attribute is now created and an edit pop-in opens with the attribute parameters depending on the attribute type.
+
+![image](../img/ReferenceEntity_AddAttribute.png)
```

Comment:
Je mettrais l'image avant la phrase "the attribute is now created and an edit pop-in opens...", parce que du coup je m'attendais à voir la pop-in. Qu'en penses-tu?

Reply:
ah yes